examples/node/overview.rs

```diff
@@ -13,7 +14,7 @@ pub fn overview(node: &LightningNode, words: &mut dyn Iterator<Item = &str>) ->
         .unwrap_or("10")
         .parse()
         .context("Number of payments should be a positive integer number")?;
```

Comment:
```suggestion
        .context("Number of movements should be a positive integer number")?;
```

Reply:
Same for variable `number_of_payments` on line 12

Reply:
Applied in [c94b054](https://github.com/getlipa/lipa-lightning-lib/pull/868/commits/c94b054cbdd4b54ad373a221c9c2edb42ce0adc5)